### ISSUE TITLE
Fix link anchors and other issues in readthedocs

### ISSUE THREAD

**Issue by heuermh** [bigdatagenomics/adam#1822]
Some link anchors and other formatting styles were broken after conversion to rst.

File pull requests against this issue for these and other issues with the new documentation hosted at http://adam.readthedocs.io/en/latest/.

E.g.

```
a known variation database (e.g., dbSNP). {#known-snps}
See candidate generation and realignment. {#known-indels}
While the `transformAlignments <#transformAlignments>`__ command
```

Regarding Pygments styles, I prefer [lovelace](http://pygments.org/demo/6682259/?style=lovelace) and [arduino](http://pygments.org/demo/6682259/?style=arduino) over the default.

**Comment by antonkulaga** [comment]
You have a small formating issue at http://adam.readthedocs.io/en/latest/architecture/overview/ with \end{itemize}

**Comment by antonkulaga** [comment]
at http://adam.readthedocs.io/en/latest/api/joins/ it is better to separate right and inner by the line on the pic

**Comment by heuermh** [comment]
> You have a small formating issue at http://adam.readthedocs.io/en/latest/architecture/overview/ with \end{itemize}

Thanks, I think we fixed that in #1827 already.

> at http://adam.readthedocs.io/en/latest/api/joins/ it is better to separate right and inner by the line on the pic

Good suggestion, thanks again.

**Comment by heuermh** [comment]
Fixed by #1827
